File: www/kd/ka.pl
#!/usr/bin/perl

use strict;
use warnings;
use CGI;

my $cgi = CGI->new;

print("Content-Type: text/plain\r\n");
print("Connection: keep-alive\r\n\r\n");
# print("Content-Length: 12\r\n\r\n");

print("PERL is not dead (yet).");

File: www/kd/ka.py
#!/usr/bin/python
from __future__ import print_function

import sys
import os

import cgi
import cgitb

print("Content-Type: text/plain", end="\r\n")
print("Connection: keep-alive", end="\r\n")
print("Content-Length: 222", end="\r\n\r\n")

print ("PYTHON is not dead (yet).", end="")